NetworkPkg/Library/DxeNetLib/DxeNetLib.c

```diff
@@ -144,7 +144,7 @@ GLOBAL_REMOVE_IF_UNREFERENCED VLAN_DEVICE_PATH  mNetVlanDevicePathTemplate = {
 // If your platform needs to use a specific algorithm for the random number
 // generator, then you should modify this array.
 //
-GLOBAL_REMOVE_IF_UNREFERENCED EFI_GUID  *mSecureHashAlgorithms[] = {
+STATIC GLOBAL_REMOVE_IF_UNREFERENCED EFI_GUID  *mSecureHashAlgorithms[] = {

```

Comment:
Use `static` for this and an improvement is to prefix the name of the global with the module/lib base name. 

`mDxeNetLibSecureHashAlgorithms`

Reply:
Ah, yes, sorry, the STATIC -> static thing slipped my mind when reviewing.

As for the name ... it is my general preference to keep file-local identifiers outside of global naming conventions. But we currently lack an indicator of file-local scope, as opposed to module-local(m), or global(g).

Reply:
Fixed in latest update.